core/templates/dev/head/pages/exploration_player/HelpCardService.js

```diff
@@ -0,0 +1,42 @@
+// Copyright 2017 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Service to track the information about Help Card
+ */
+
+oppia.factory('HelpCardService', [
+  function() {
+    var localHelpCard = {
```

Comment:
It would be nice if you could make this a domain object instead of a raw JS dict.